base/uk.ac.stfc.isis.ibex.ui.configserver/src/uk/ac/stfc/isis/ibex/ui/configserver/dialogs/MultipleConfigsSelectionDialog.java

```diff
@@ -124,20 +124,27 @@ protected void createSelection(Composite container) {
         } else {
             names = ConfigInfo.namesWithoutCurrent(available).toArray(new String[0]);
         }
+         
 		Arrays.sort(names, String.CASE_INSENSITIVE_ORDER);
 		
 		setItems(names, compOrConfigNamesWithFlags);
 		
-		Group group = new Group(container, SWT.SHADOW_IN);
-		group.setLayout(new RowLayout(SWT.HORIZONTAL));
-		        
-	    Label messageImageLabel = new Label(group, SWT.NONE);
-            messageImageLabel.setImage(JFaceResources.
-                getImage(DLG_IMG_MESSAGE_WARNING)); 
-
-        Label messageLabel = new Label(group, SWT.NONE);
-        messageLabel.setFont(JFaceResources.getDialogFont());
-        messageLabel.setText("Represents Protected " + getTypeString());
+		// show protected configuration message only if relevant (the list has protected configurations)
+		Boolean hasProtectedConfigs = ConfigInfo.hasProtectedConfigs(available);
+		if (hasProtectedConfigs)
+		{
+			Group group = new Group(container, SWT.SHADOW_IN);
+			group.setLayout(new RowLayout(SWT.HORIZONTAL));
+			        
+			
+		    Label messageImageLabel = new Label(group, SWT.NONE);
+	            messageImageLabel.setImage(JFaceResources.
```

Comment:
Checkstyle may complain about the indentation here. The call to setImage should be indented at the same level as the Label declaration